frontend/src/metabase/css/core/text.css

```diff
@@ -61,7 +61,7 @@
     .xl-text-right { text-align: right; }
 }
 
-.text-uppercase, :local(.text-uppercase) { text-transform: uppercase; }
+.text-uppercase, :local(.text-uppercase) { text-transform: uppercase; letter-spacing: 0.8px;}
```

Comment:
So, I tried `ems` but I think it's actually less predictable/desirable. It results in too much spacing for larger all-caps headings, like the ones in the forms of the admin panel.